inc/server/class-form-server.php

```diff
@@ -58,8 +58,39 @@ public function register_routes() {
 				),
 			)
 		);
+
+
+		register_rest_route(
+			$namespace,
+			'/mailchimp',
```

Comment:
You can use one 'newsletter' route and pass the provider as an argument so we don't need to make a new rest route each time we want to integrate with a new service. @Soare-Robert-Daniel